lib/Models/Cesium.js

```diff
@@ -423,6 +423,138 @@ Cesium.prototype.computePositionOnScreen = function(position, result) {
     }
 };
 
+/**
+ * Updates the order of layers on the Leaflet map to match the order in the Now Viewing pane.
+ */
+Cesium.prototype.updateLayerOrder = function() {
```

Comment:
The name of this is a bit misleading in the context of Cesium, right?  Because it doesn't update layer order in general, it only moves `keepOnTop` layers to the top.